templates/Web/_composition/Flask/Page.Flask.AddSampleData.ForAllExceptList/Page.Flask.AddSampleData.ForAllExceptList/server/sampleData.py

```diff
@@ -0,0 +1,5 @@
+sampleData = {}
+
+# This class holds sample data used by some generated pages to show how they can be used.
+# Todo Web Template Studio: Delete this file once your app is using real data.
```

Comment:
I am nitpicking here, "Todo" is all capitalized for the node template. Is "Todo" the convention in python?

Reply:
Yeah, it should be capitalized. Corrected it